src/pages/api/mail/contact.ts

```diff
@@ -1,112 +1,108 @@
-    } = req.body
-    const aditionalInformationHeader =
-      discord || telegram || github || projectLink || aditionalInformation
-        ? 'Aditional information'
-        : ''
-    const discordHandle = discord ? `Discord handle: ${discord}` : ''
-    const telegramHandle = telegram ? `Telegram handle: ${telegram}` : ''
-    const githubHandle = github ? `Github Profile: ${github}` : ''
-    const projectLinkText = projectLink ? `Project Link: ${projectLink}` : ''
-    const aditionalInformationText = aditionalInformation
-      ? `Additional Information given by the client: ${aditionalInformation}`
-      : ''
-    const mailData = {
-      from: process.env.GMAIL_ACCOUNT as string,
-      to: process.env.EMAIL_RECIPIENT as string,
-      subject: `Project Request from: ${firstName} | ${projectName}`,
-      text: `Who requested: 
-      First Name: ${firstName} 
-      E-mail: ${email}
-      Project Name: ${projectName}
-      Project Description: ${projectDescription}
-      Type of services: ${services}
-      Budget Range: ${budget}
-      Deadline(yyyy-mm-dd): ${deadline}
+    const submitData: Submit = req.body
+    const { success } = SubmitSchema.safeParse(submitData)
+    if (success) {
+      const {
+        firstName,
+        email,
+        projectName,
+        projectDescription,
+        services,
+        budget,
+        deadline,
+        discord,
+        telegram,
+        github,
+        projectLink,
+        aditionalInformation,
+      } = submitData
 
-      ${aditionalInformationHeader}
-      ${discordHandle}
-      ${telegramHandle}
-      ${githubHandle}
-      ${projectLinkText}
-      ${aditionalInformationText}
-      `,
-    }
-    const date = new Date()
-    const sheetsData = {
-      timeStamp: date.toUTCString(),
-      whoRequested: firstName,
-      emailContact: email,
-      projectName: projectName,
-      projectDescription: projectDescription,
-      typeOfServices: services.toString(),
-      budgetRange: budget,
-      deadline: deadline,
-      discordHandle: discord ? discord : '',
-      githubHandle: github ? github : '',
-      telegramHandle: telegram ? telegram : '',
-      projectLink: projectLink ? projectLink : '',
-      aditionalInformationText: aditionalInformation ? aditionalInformation : ''
-    }
-    try {
-      mailingClient.sendMail(mailData, (err) => {
-        if (err) {
-          console.log(err)
-          res.status(500).json({
-            success: false,
-            body: err,
-          })
-        } else {
-          res.status(200).json({
-            success: true,
-          })
-        }
-      })
-      appendSpreadsheet(sheetsData)
-    } catch (err) {
-      console.log(err)
-      res.status(400).json({
+      const aditionalInformationHeader =
+        discord || telegram || github || projectLink || aditionalInformation
+          ? 'Aditional information'
+          : ''
+      const discordHandle = discord ? `Discord handle: ${discord}` : ''
```

Comment:
To keep it DRY, I recommend creating a helper function that takes the optional data variable and its message and returns an empty string, and replaces all of these ternary operators.

something like that
```
// Helper function that returns an empty string if the optional data is falsy
    const handleOptionalData = (optionalData: string, message: string) => optionalData ? message : ''

// instead of 
const discordHandle = discord ? `Discord handle: ${discord}` : ''

// replacing for
const discordHandle = handleOptionalData(discord, `Discord handle: ${discord}`)

```